api/src/application/email/drivers/mailgun.js

```diff
@@ -58,7 +59,7 @@ module.exports = function () {
 
         mailgunClient.messages().send(mail, (error, body) => {
           if (error) {
-            console.log(error, mail);
+            logger.info(error, mail);
```

Comment:
seems like this should be `logger.error` - and what happens with the second argument? I don't remember

Reply:
I thought the same, but left it at info to stick with the original implementation. Although, I can change it. 

This is how it looks: 

`api_1       | info: Message from=no-reply@scrubbed.com, to=me@scrubbed.com, subject=Verify your email address, text=, html=Click this link to verify your email on : http://localhost:9000/user/email-verify?client_id=731f98f5-a2b6-47ea-b5ee-55ec84ec0c3f&redirect_uri=https%3A%2F%2Fsso-ient.test%3A30%2F&token=e72bab1d-109a-4a31-824d-af92449f3a0a`